.github/workflows/ci-container-image.yml

```diff
@@ -1,9 +1,10 @@
-name: Build and push latest image to container registry
+name: container-image
 
 on:
   push:
-    branches:
-      - main
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
```

Comment:
Trying to understand this? Doesn't this run on every pull request to main then?

Reply:
This is goal of this patch, to run it tor every new PR.

Currently container build runs only when patch is commited to main. (or via schedule)